app/services/manage_action.rb

```diff
@@ -32,5 +33,9 @@ def queue_message
       }
     }
   end
+
+  def generate_referring_user
+    @params[:referring_user] = "/rest/v1/user/#{actionkit_user_id(@params.delete(:referring_akid))}/" if @params.has_key? :referring_akid
```

Comment:
what is this about? we pass referring user as a path on... ak? is this ak specific? is this the place for ak specific code?